chameleon/keystone_auth.py

```diff
@@ -162,7 +162,7 @@ def regenerate_tokens(request, password):
     email = request.user.email
     tokens = {}
 
-    for region in settings.OPENSTACK_AUTH_REGIONS.keys():
+    for region in list(settings.OPENSTACK_AUTH_REGIONS.keys()):
```

Comment:
This `list()` optimization is something that `2to3` does just to be safe, but in most cases it is not necessary, as in this case. Fine to leave it in though, as it doesn't hurt anything really for us.